templates/workspace_services/guacamole/terraform/web_app.tf

```diff
@@ -192,3 +196,20 @@ resource "azurerm_private_endpoint" "guacamole" {
     private_dns_zone_ids = [data.azurerm_private_dns_zone.azurewebsites.id]
   }
 }
+
```

Comment:
L67 (the identity) I guess it should be removed. since its not used anymore

Reply:
It is used by the guacamole, when fetching the secrets from kv

Reply:
Lines 208 till the end

Reply:
Lines 208 till the end